css/style.css

```diff
@@ -6,15 +6,13 @@
 /* general */
 
 * {
-  -webkit-box-sizing: border-box;
-  -moz-box-sizing: border-box;

```

Comment:
I wonder if we don't leave these in just to be safe? The only point would be for very rare backwards compatibility cases.

Reply:
Sure, according to [caiuse](https://caniuse.com/mdn-css_properties_box-sizing_border-box), it has good support; we shouldn't worry about that small percentage.